test/utils.py

```diff
@@ -16,7 +16,9 @@
 ###
 
 import yaml
-from mock import mock, patch
+import mock
+
+from mock import patch
```

Comment:
We should refactor this module, we are importing patch twice.
Should we do it now or in another PR?